packages/aws-sdk-client-mock-jest/src/jestGlobals.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import './jest';
+import { AwsSdkMockMatchers } from './jestMatchers';
+
+/**
+ * Types for @jest/globals
+ */
+declare module 'expect' {
+  interface Matchers<R = void> extends AwsSdkMockMatchers<R> {}
+}
```

Comment:
Is there a reason why this can't be in `jest.ts` next to types for `@types/jest`, like before? Without having a separate import when using `@jest/globals`?

Otherwise, this would be a breaking change and the PR title must be `feat!: ...`

Reply:
https://github.com/m-radzikowski/aws-sdk-client-mock/issues/216#issuecomment-2016782568
I followed the import method of `jest-dom` as you mentioned. But I think you are right, I should not introduce a breaking change in a PR that is adding support for vitest.